extension/src/messageHandlers.js

```diff
@@ -118,7 +122,7 @@ export async function signMessageHandler(
 export async function walletMessageHandler(message, sender, sendResponse) {
   switch (message.type) {
     case 'GET_WALLETS': {
-      sendResponse(getWalletIndex())
+      sendResponse(getWalletIndex(true))
```

Comment:
why do you need to pass `true` here?

Reply:
to get the enriched version, with network and crypto in

Reply:
why isn't that the default?

Reply:
:shrug: